src/features/lecture-evaluation/components/LectureEvaluationCard/LectureEvaluationCard.module.scss

```
@@ -0,0 +1,212 @@
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  min-height: 45px;
+  padding: spacing.$space-8;
+  color: colors.$gray-100;
+  background: colors.$white;
+  border: 1px solid colors.$background-color;
+  border-radius: 6px;
+  @include typography.body-sm-design;
+  @include typography.font-weight-extrabold;
+  text-align: center;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)